modules/stanford_earth_capx/src/Plugin/migrate/process/StanfordEarthCapProfileImage.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace Drupal\stanford_earth_capx\Plugin\migrate\process;
+
+use Drupal\Core\File\FileSystemInterface;
+use Drupal\Core\StreamWrapper\StreamWrapperManagerInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\Plugin\MigrateProcessInterface;
+use Drupal\migrate\Plugin\Migration;
+use Drupal\migrate\Row;
+use Drupal\migrate_file\Plugin\migrate\process\FileImport;
+use Drupal\file\Entity\File;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use GuzzleHttp\Exception\ClientException;
+use Drupal\stanford_earth_capx\EarthCapxInfo;
+
+/**
+ * Imports an image from a Stanford Profiles CAP API url 
```

Comment:
Thanks for the verbose comment!

Reply:
Verbose comment courtesy of the migrate_file module from which this code was copied.